ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -6,7 +6,7 @@
 from geometry_msgs.msg import PoseStamped
 from geometry_msgs.msg import TwistStamped
 from styx_msgs.msg import Lane, Waypoint
-from styx_msgs.msg import TrafficLightArray
+from styx_msgs.msg import TrafficLightArray, TrafficLight
```

Comment:
I am not following, but are we actually using TrafficLight/Array here? I think we just receive index of next traffic light waypoint?